cursorless-talon-dev/src/cursorless_dev.talon

```diff
@@ -26,5 +28,5 @@ tag: user.cursorless
 {user.cursorless_homophone} update cheatsheet:
     user.private_cursorless_cheat_sheet_update_json()
 
-test snippet make <user.cursorless_target>:
-    user.private_cursorless_make_snippet_test(cursorless_target)
+playground <user.cursorless_target>:
```

Comment:
I think I agree with @josharian that we should possibly name this something else, esp since the "playground" we make will prob be more sophisticated. Could use eg "show parse", "parse tree", "tree sitter", "tree sit", etc

Reply:
`Parse tree` for both the spoken form and the action name sounds reasonable.